app/src/main/java/space/taran/arknavigator/mvp/model/repo/stats/StatsStorage.kt

```diff
@@ -0,0 +1,28 @@
+package space.taran.arknavigator.mvp.model.repo.stats
+
+import space.taran.arknavigator.mvp.model.repo.index.ResourceId
+import space.taran.arknavigator.utils.Tag
+import space.taran.arknavigator.utils.Tags
+
+sealed class StatsEvent(val categories: List<StatsCategory>) {
```

Comment:
Is `List` argument necessary for future support of `tag-queried-n` and `tag-queried-ts` metrics?

Reply:
No, but this feature may come in handy in the future. Since one event can trigger several metrics